paparazzi/paparazzi-gradle-plugin/src/test/projects/locale-qualifier-xml/src/main/res/values-en-rGB/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="color">Colour</string>
```

Comment:
two space indents on all resources.  surprised spotless didn't catch this.

Reply:
Interesting. AS also didn't reformat them with editorconfig. Reformatted here:1de4494f47886bff78fd3f3a92714a8a6e080c59